previews/template_preview.py

```diff
@@ -24,7 +27,102 @@ class SynonymeForm(forms.Form):
     )
 
 
+class PinPointForm(forms.Form):
+    action = forms.ChoiceField(
+        label="Action",
+        help_text="Sélectionnez une action",
+        initial="reparer",
+    )
+    avec_bonus = forms.BooleanField(
+        label="Avec bonus",
+        help_text="Avec bonus",
+        initial=False,
+    )
+    carte = forms.BooleanField(
+        label="Carte",
+        help_text="Carte",
+        initial=True,
+    )
+    carte_config = forms.ChoiceField(
+        label="Carte config",
+        help_text="Carte config",
+        required=False,
+        choices=[("", "")] + list(CarteConfig.objects.all().values_list("id", "nom")),
+        initial="",
+    )
+    acteur_type = forms.ChoiceField(
+        label="Acteur type",
+        help_text="Action list",
+        required=False,
+        choices=[("", "")]
+        + list(ActeurType.objects.all().values_list("code", "libelle")),
```

Comment:
suggestion pour être plus _pythonique_ 
```suggestion
        choices=[("", ""),
        *list(ActeurType.objects.all().values_list("code", "libelle"))],
```